CHANGELOG.rst

```diff
@@ -1,7 +1,16 @@
 .. _changelog:
-
 0.55.0
 ------
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+- Set resource pool/class for a session launcher 2.0  (`#3196 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3196>`_)
```

Comment:
(The docs formatter test was breaking because of a missing newline on line 2, so I added that too)

```suggestion

0.55.0
------
User-Facing Changes
~~~~~~~~~~~~~~~~~~~

**✨ Improvements**

- Support setting a default resource class for a session launcher in Renku 2.0  (`#3196 <https://github.com/SwissDataScienceCenter/renku-ui/pull/3196>`_)
```